documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -137,3 +144,10 @@ Read our troubleshooting steps, tips, and tricks here: xref:display.adoc#trouble
 * Outer dimensions: 192.96 × 110.76mm
 * Viewable area: 154.08 × 85.92mm
 * https://datasheets.raspberrypi.com/display/7-inch-display-mechanical-drawing.pdf[Download mechanical drawing (PDF)]
+
+[.whitepaper, title="Touch Display Enclosure Mechanical Design", subtitle="", link=https://pip.raspberrypi.com/categories/685-whitepapers-app-notes/documents/RP-003473-WP/Touch-Display-Enclosure-Mechanical-Design.pdf]
```

Comment:
Not sure we need to link to this one, and if we do it should be from https://www.raspberrypi.com/documentation/accessories/display.html rather than here, surely?